driver.sh

```diff
@@ -44,34 +72,29 @@ build_linux() {
     git fetch --depth=1 origin master
     git reset --hard origin/master
   fi
-  export ARCH=arm64
-  export CROSS_COMPILE=aarch64-linux-gnu-
   # Only clean up old artifacts if requested, the Linux build system
   # is good about figuring out what needs to be rebuilt
   [[ -n "${cleanup:-}" ]] && mako_reactor mrproper
   mako_reactor defconfig
-  mako_reactor Image.gz
+  mako_reactor $config
+  mako_reactor $image_name
   cd "$OLDPWD"
 }
 
 boot_qemu() {
-  local kernel_image=linux/arch/${ARCH}/boot/Image.gz
-  local rootfs=images/${ARCH}/rootfs.ext4
+  local kernel_image=linux/arch/${ARCH}/boot/$image_name
```

Comment:
mixing `${ARCH}` and `$image_name` in the same statement :-1: Please pick one and be consistent.

Reply:
I acknowledge I have not been consistent in one vs the other in the script, but mixing in the same statement makes the inconsistency much more obvious.

Reply:
Hmm what's the issue exactly? They're guaranteed to be set together in `set_variables`.

`$image_name` is also needed for the `make` command in `build_linux` but I guess I can make the `ARCH` a wild card.

Reply:
Sorry, I meant:

```diff
- local kernel_image=linux/arch/${ARCH}/boot/$image_name
+ local kernel_image=linux/arch/${ARCH}/boot/${image_name}
```

both `ARCH` and `image_name` are variables, right?

Reply:
OH yes duh... I'll personally use braces on all my variables, I think it makes them stand out a bit but I'll use whatever style you/others prefer, I'm flexible.